common/src/Microsoft.Azure.IIoT.Messaging.SignalR/src/Services/SignalRServiceHost.cs

```diff
@@ -45,6 +45,8 @@ public class SignalRServiceHost : IIdentityTokenGenerator, IEndpoint,
             }).Build();
             Resource = !string.IsNullOrEmpty(config.SignalRHubName) ?
                 config.SignalRHubName : "default";
+            _renewHubTimer = new Timer(RenewHubTimer_ElapesedAsync);
```

Comment:
Please add a TODO: To remove this when signal r service SDK is fixed.

Reply:
done